docroot/themes/contrib/civictheme/includes/utilities.inc

```diff
@@ -753,3 +753,17 @@ function _civictheme_feature_optout_flags() {
     'components.link.email' => t('Email links processing'),
   ];
 }
+
+/**
+ * Checks if a file path has the SVG extension.
+ *
+ * @param string $file_path
+ *   File path to check.
+ *
+ * @return bool
+ *   TRUE if the file path has SVG extension, FALSE otherwise.
```

Comment:
`TRUE if the file is SVG, FALSE otherwise.`
We do not want to disclose the implementation in comments.

Reply:
but we may no need this function at all since the SVG file can be passed to the `<img>` tag as a `src` attribute value.